include/deal.II/lac/schur_complement.h

```diff
@@ -0,0 +1,498 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2015 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii__schur_complement_h
+#define dealii__schur_complement_h
+
+#include <deal.II/base/config.h>
+#include <deal.II/base/exceptions.h>
+#include <deal.II/lac/vector_memory.h>
+#include <deal.II/lac/linear_operator.h>
+#include <deal.II/lac/packaged_operation.h>
+
+#ifdef DEAL_II_WITH_CXX11
+
+DEAL_II_NAMESPACE_OPEN
+
+/**
+ * @name Creation of a LinearOperator related to the Schur Complement
+ */
+//@{
+
+/**
+ * @relates LinearOperator
+ *
+ * Returns a LinearOperator that performs the operations
+ * associated with the Schur complement. There are two additional
+ * helper functions, condense_schur_rhs() and postprocess_schur_solution(), that are likely
+ * necessary to be used in order to perform any useful tasks in linear
+ * algebra with this operator.
+ *
+ * We construct the definition of the Schur complement in the following way:
+ *
+ * Consider a general system of linear equations that can be
+ * decomposed into two major sets of equations:
+ * @f{eqnarray*}
+  \mathbf{K}\mathbf{d} = \mathbf{f}
+  \quad \Rightarrow\quad
+  \left(\begin{array}{cc}
+    A & B \\ C & D
+  \end{array}\right)
+  \left(\begin{array}{cc}
+    x \\ y
+  \end{array}\right)
+  =
+  \left(\begin{array}{cc}
+    f \\ g
+  \end{array}\right),
+ * @f}
+ * where \f$ A,B,C,D \f$  represent general subblocks of the matrix
```

Comment:
You don't need to use `\f`. We automatically expand the dollar sign to the right sequence. Just write as if you used latex.